package.json

```diff
@@ -1,29 +1,32 @@
 {
-  "name": "randbot",
-  "version": "1.0.0",
-  "private": true,
-  "author": "hash-gaming",
-  "description": "Mostly productive but sometimes trolly",
-  "scripts": {
-    "linter": "eslint ."
-  },
-  "dependencies": {
-    "hubot": "2.19.0",
-    "hubot-diagnostics": "0.0.2",
-    "hubot-hello": "0.0.1",
-    "hubot-help": "0.2.2",
-    "hubot-pugme": "0.1.1",
-    "hubot-redis-brain": "0.0.4",
-    "hubot-rules": "0.1.2",
-    "hubot-scripts": "2.17.2",
-    "hubot-slack": "4.4.0"
-  },
-  "engines": {
-    "node": ">=8.0.0"
-  },
-  "devDependencies": {
-    "eslint": "4.8.0",
-    "eslint-config-airbnb-base": "12.0.2",
-    "eslint-plugin-import": "2.7.0"
-  }
+    "name": "randbot",
+    "version": "1.0.0",
+    "private": true,
+    "author": "hash-gaming",
+    "description": "Mostly productive but sometimes trolly",
+    "scripts": {
+        "linter": "eslint .",
+        "local-server-start": "docker-compose up --build --force-recreate",
+        "local-server-stop": "docker rm $(docker ps -a -q)",
+        "start": "./bin/hubot --name randbot"
+    },
+    "dependencies": {
+        "hubot": "2.19.0",
+        "hubot-diagnostics": "0.0.2",
+        "hubot-hello": "0.0.1",
+        "hubot-help": "0.2.2",
+        "hubot-pugme": "0.1.1",
+        "hubot-redis-brain": "0.0.4",
+        "hubot-rules": "0.1.2",
+        "hubot-scripts": "2.17.2",
+        "hubot-slack": "4.4.0"
+    },
+    "engines": {
+        "node": ">=8.0.0"
+    },
+    "devDependencies": {
+        "eslint": "4.8.0",
+        "eslint-config-airbnb-base": "12.0.2",
+        "eslint-plugin-import": "2.7.0"
+    }
```

Comment:
looks like my linter did the 4 space

Reply:
Yeah, and anytime we include a new npm package or do anything with `npm`, it's going to rewrite it in two spaces so can we go back to two?

Reply:
🎉